lib/src/auth/auth.dart

```diff
@@ -1,18 +1,99 @@
-/// ref https://developer.qiniu.com/kodo/manual/1644/security
+import 'dart:convert';
+import 'package:meta/meta.dart';
+import 'package:crypto/crypto.dart';
+import './put_policy.dart';
+
+class TokenInfo {
+  final String accessKey;
+  final PutPolicy putPolicy;
+  const TokenInfo(this.accessKey, this.putPolicy);
+}
+
+/// 提供用于鉴权的相关功能
+///
+/// 安全机制文档 ref https://developer.qiniu.com/kodo/manual/1644/security
 class Auth {
-  final String ak;
-  final String sk;
-  Auth(this.ak, this.sk);
+  /// 鉴权所需的 [AccessKey]
+  ///
+  /// 如何生成以及查看请参阅：
+  /// http://developer.qiniu.com/article/developer/security/index.html
```

Comment:
![image](https://user-images.githubusercontent.com/7148809/93455893-8803d680-f90f-11ea-9a51-6ff0468cbcbf.png)
```suggestion
  /// 如何生成以及查看请参阅：
  
  /// http://developer.qiniu.com/article/developer/security/index.html
```

Reply:
换不换行 出来文档不一样？

Reply:
> 换不换行 出来文档不一样？

确实不一样

![image](https://user-images.githubusercontent.com/22045569/93546311-1ec6a680-f995-11ea-949f-e4964f488e58.png)